.ci/load/Jenkinsfile

```diff
@@ -91,7 +92,14 @@ pipeline {
                                         echo 'Switching to requested version'
                                         sh(script: "git checkout v${params.apm_version}")
                                         echo 'Building agent'
-                                        sh(script: './mvnw  clean install -DskipTests=true -Dmaven.javadoc.skip=true')
+                                        withEnv(["JAVA_HOME=${env.HUDSON_HOME}/.java/java11"]){
+                                            withEnv(["JAVACMD=${env.JAVA_HOME}/bin/java", "PATH=${env.JAVA_HOME}/bin:$ORIG_PATH"]){
```

Comment:
I didn't understand the trick of nester withEnv steps, but I just tested and it seems there is no other way.

Regarding the `PATH` variable, can we skip `ORIG_PATH` and use `"JAVA_HOME+PATH=${env.JAVA_HOME}/bin"`? See [with-env](https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#withenv-set-environment-variables) for a better explanation of what I mean :)

Reply:
Here I'd still prefer to use `ORIG_PATH` because I want to ensure that we have a clean path without Java explicitly defined anywhere on it. If we prepend to the current path as you suggest, we'll be prepending to what we previously set in L77.